class/fluentbit.yml

```
@@ -2,7 +2,7 @@ parameters:
   kapitan:
     dependencies:
       - type: helm
-        output_path: dependencies/fluentbit/helmcharts/fluent-bit-${fluentbit:charts:fluent-bit:version}
+        output_path: dependencies/${_instance}/helmcharts/fluent-bit-${fluentbit:charts:fluent-bit:version}
```

This is incorrect, there won't be a `dependencies/<instance name>` for component instances.

If anything, you should update this to use `${_base_directory}`:

```suggestion
        output_path: ${_base_directory}/helmcharts/fluent-bit-${fluentbit:charts:fluent-bit:version}
```